tests/kafkatest/services/templates/kafka.properties

```diff
@@ -20,6 +20,8 @@ port=9092
 #host.name=localhost
 advertised.host.name={{ node.account.hostname }}
 #advertised.port=<port accessible by clients>
+listeners=PLAINTEXT://:9092,SSL://:9093
```

Comment:
Yes, agreed that any new tests should be in a separate PR. We don't have a definite list of tests to be written, although that is something we need to do.

I was just trying to explain that there will be cases where we'd want to have a mixed SSL and non-SSL connections. We can handle those when the need arises, if it's easier.